packages/node_modules/@ciscospark/spark-core/src/config.js

```diff
@@ -10,6 +10,8 @@ const IDBROKER_BASE_URL = process.env.IDBROKER_BASE_URL || 'https://idbroker.web
 
 export default {
   maxAppLevelRedirects: 10,
+  maxLocusRedirects: 5,
+  LOCUS_REDIRECT_ERROR: 2000002,
```

Comment:
Did you mean to make this a user configurable option? Seems more like a constant that shouldn't be changeable by the developer.